src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -182,15 +184,21 @@ def update_rule(nlcd, soil, count, survey):
 
     def after_rule(count, survey):
         nlcd_names = [v[1] for v in NLCD_MAPPING.values()]
+        nlcd_keys = NLCD_MAPPING.keys()
```

Comment:
See [Python Documentation](https://docs.python.org/2/library/stdtypes.html#dict.items) which states that calles to `values` and `keys` (among others) will return in the same order, assuming no intervening modifications have been made.